extra/swayimgrc.5

```diff
@@ -102,11 +102,15 @@ Default image scale, valid modes are:
 .IP "\fBkeep_zoom\fR\fR = \fI[yes|no]\fR"
 Keep absolute zoom across images, \fIno\fR by default.
 .\" ----------------------------------------------------------------------------
+.IP "\fBkeep_position\fR\fR = \fI[yes|no]\fR"
+Keep absolute position across images, \fIno\fR by default.
+.\" ----------------------------------------------------------------------------
 .IP "\fBposition\fR = \fIPOSITION\fR"
 Initial image position, valid modes are:
 .nf
 \fItop\fR: move image to top and center by width;
-\fIcenter\fR: center by width and height (default);
+\fIcenter\fR: center by image width and height;
+\fIabsolutecenter\fR: center by window width and height (default);
```

Comment:
This looks confusing. What the difference between center and absolute center? Is the first center relative now?

Reply:
The way I saw it is that positions are based on the image. However, zooming was based on the center of the window rather than the center of the image. Since this anchoring thing was based on the image instead of the window, I thought it would be a good idea to change zooming to be based on the image center rather than the window center.

I can rename absolutecetner->windowcenter, that might make it more clear; or I can have center->imagecenter and revert the other center. Whatever you think is more clear.

Reply:
Removed absolute center. Zooming anchored to various parts of the image feels unintuitive and I feel silly for implementing it.